### ISSUE TITLE
\mathord

### ISSUE THREAD

**Issue by hbghlyj** [ronkok/Temml#53]
**Example**
```tex
\begin{gather}
\nabla\varphi\\
\mathord\nabla\varphi
\end{gather}
```
|In [Temml](https://temml.org/#%5Cbegin%7Bgather%7D%0A%5Cnabla%5Cvarphi%5C%5C%0A%5Cmathord%5Cnabla%5Cvarphi%0A%5Cend%7Bgather%7D) `\mathord\nabla` is italic|In [LaTeX](https://texlive.net/run?%5Cdocumentclass%5Bpreview%5D%7Bstandalone%7D%0A%5Cusepackage%7Bamsmath%7D%0A%5Cbegin%7Bdocument%7D%0A%5Cbegin%7Bgather%7D%0A%5Cnabla%5Cvarphi%5C%5C%0A%5Cmathord%5Cnabla%5Cvarphi%0A%5Cend%7Bgather%7D%0A%5Cend%7Bdocument%7D) `\mathord\nabla` is normal|
|--|--|
|<img src="https://github.com/ronkok/Temml/assets/53823634/a8e33c9b-d611-409b-9aa1-7570cab8f057" height="50">|![1](https://github.com/ronkok/Temml/assets/53823634/4778b045-ffe3-483e-a34b-b0af08862205)|

In LaTeX, the \mathord\nabla command is expected to produce a normal (non-italic) nabla symbol. However, in Temml, this symbol is being placed inside a &lt;mi> element, which results in it being rendered in an italicized form.

I suggest place characters that are typically normal in a &lt;mi> element with the attribute `mathvariant="normal"`. This would ensure that the nabla symbol appears in its normal (non-italic) form, consistent with LaTeX conventions.

**Comment by hbghlyj** [comment]
Example
```tex
\mathord\Delta
```
The result is `<mi><mi mathvariant="normal">Δ</mi></mi>`

**Comment by ronkok** [comment]
In the next day or two, I will publish an update that fixes this specific problem. However, I cannot completely fix the larger issue of invalid MathML inside a `<mi>` element.

In LaTeX, it is possible to wrap a `\mathord` around an expression. Example: `\mathord{\frac a b}`. That doesn't make much sense, but it is possible. I can think of no way to express such a thing in MathML but to wrap the expression in an `<mi>` element.  The MathML spec says that is invalid, but I cannot think of any alternative.

Anytime `\mathord` is wrapped around a single character, Temml will emit an `<mi>` containing just that character. That is valid MathML. But for larger expressions, I think the invalid MathML is here to stay.

**Comment by ronkok** [comment]
This instance of the invalid `<mi>` problem is fixed by release v0.10.22.
